tabbycat/templates/tables/Popover.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <v-touch class="touch-target" ref="container" v-on:tap="togglePopOver">
```

Comment:
I *think* that when this was changed from being a component to a standard div, the `$el` reference in `createPopper(this.$refs.container.$el` was no longer valid.